server/models/procedures/trial_balance.sql

```diff
@@ -96,91 +96,107 @@ CREATE PROCEDURE TrialBalanceErrors()
     FROM posting_journal AS pj JOIN stage_trial_balance_transaction AS temp
       ON pj.record_uuid = temp.record_uuid
+      JOIN account AS ac ON ac.id = pj.account_id
     JOIN debtor_group dg ON pj.account_id = dg.account_id
     WHERE dg.locked = 1 GROUP BY pj.record_uuid;
 
   -- check that all accounts are unlocked
   INSERT INTO stage_trial_balance_errors
-    SELECT pj.record_uuid, MAX(pj.trans_id), 'POSTING_JOURNAL.ERRORS.LOCKED_ACCOUNT' AS code
+    SELECT pj.record_uuid, MAX(pj.trans_id), ac.id, CONCAT(ac.number,' - ', ac.label)  as error_description, 'POSTING_JOURNAL.ERRORS.LOCKED_ACCOUNT' AS code
     FROM posting_journal AS pj JOIN stage_trial_balance_transaction AS temp
       ON pj.record_uuid = temp.record_uuid
+      JOIN account AS ac ON ac.id = pj.account_id
     JOIN account a ON pj.account_id = a.id
     WHERE a.locked = 1 GROUP BY pj.record_uuid;
 
   -- check that users are active (no deactivated users)
   INSERT INTO stage_trial_balance_errors
-    SELECT pj.record_uuid, MAX(pj.trans_id), 'POSTING_JOURNAL.ERRORS.DEACTIVATED_USER' AS code
+    SELECT pj.record_uuid, MAX(pj.trans_id), ac.id,  u.display_name  as error_description, 'POSTING_JOURNAL.ERRORS.DEACTIVATED_USER' AS code
     FROM posting_journal AS pj JOIN stage_trial_balance_transaction AS temp
       ON pj.record_uuid = temp.record_uuid
+      JOIN account AS ac ON ac.id = pj.account_id
     JOIN user u ON pj.user_id = u.id
     WHERE u.deactivated = 1 GROUP BY pj.record_uuid;
 
   -- check that all transactions are balanced
   INSERT INTO stage_trial_balance_errors
-    SELECT pj.record_uuid, MAX(pj.trans_id), 'POSTING_JOURNAL.ERRORS.UNBALANCED_TRANSACTIONS' AS code
+    SELECT pj.record_uuid, MAX(pj.trans_id), ac.id, pj.trans_id  as error_description, 'POSTING_JOURNAL.ERRORS.UNBALANCED_TRANSACTIONS' AS code
     FROM posting_journal AS pj JOIN stage_trial_balance_transaction AS temp
       ON pj.record_uuid = temp.record_uuid
+      JOIN account AS ac ON ac.id = pj.account_id
     GROUP BY pj.record_uuid
     HAVING ROUND(SUM(pj.debit_equiv), 2) <> ROUND(SUM(pj.credit_equiv), 2);
 
   -- check that all transactions have two or more lines
   INSERT INTO stage_trial_balance_errors
-    SELECT pj.record_uuid, MAX(pj.trans_id), 'POSTING_JOURNAL.ERRORS.SINGLE_LINE_TRANSACTION' AS code
+    SELECT pj.record_uuid, MAX(pj.trans_id), ac.id, pj.trans_id  as error_description, 'POSTING_JOURNAL.ERRORS.SINGLE_LINE_TRANSACTION' AS code
     FROM posting_journal AS pj JOIN stage_trial_balance_transaction AS temp
       ON pj.record_uuid = temp.record_uuid
+      JOIN account AS ac ON ac.id = pj.account_id
     GROUP BY pj.record_uuid
     HAVING COUNT(pj.record_uuid) < 2;
 
-  SELECT DISTINCT BUID(record_uuid) AS record_uuid, trans_id, code FROM stage_trial_balance_errors ORDER BY code, trans_id;
+  SELECT DISTINCT BUID(record_uuid) AS record_uuid, trans_id, account_id, account_label, code FROM stage_trial_balance_errors ORDER BY code, trans_id;
```

Comment:
Does `account_label` still exist?  It looks like it should be `error_description`.